src/items.rs

```diff
@@ -790,6 +737,55 @@ fn format_impl_ref_and_type(context: &RewriteContext,
     }
 }
 
+// Returns false if failed to update result: then, try using multiline.
+fn format_trait_ref_then_update_result(context: &RewriteContext,
```

Comment:
Why update result rather than taking an offset and returning a substring (well, an Option)?